BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Controllers/UsersController.cs

```diff
@@ -43,6 +43,7 @@ public ActionResult Details2(int? id)
                 return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
             }
             UserAndPosts user = new UserAndPosts();
+            User temp = db.Users.Find(id);
```

Comment:
Are you using temp in your code? Because i cant see it in use in UsersController.cs

Reply:
Ah your right. I will remove it

Reply:
Wait hold on, that's an outdated file. Most recent commit has that removed

Reply:
Oh, let me double check

Reply:
You are right. Resolving this conversation